modules/admin-ui-picard/app/src/components/Login.js

```diff
@@ -30,7 +33,13 @@ const Login = () => {
     const [displayMenuLang, setMenuLang] = useState(false);
 
     // TODO: This is only a placeholder. Implement actual error handling of login
-    let isError = true;
+    let isError = false;
```

Comment:
Shouldn't this be using useState?
```suggestion
    const [isError, setError] = useState(false);
```